atlast_sc/data.py

```diff
@@ -137,7 +137,7 @@ def __post_init__(self):
 
     # Sideband Ratio
```

Comment:
Could we add a comment here?
```suggestion
    # Sideband Ratio - 0 for SSB and 2SB receivers, 1 for DSB receivers
```